google-datacatalog-kafka-connector/tests/google/datacatalog_connectors/kafka/sync/datacatalog_synchronizer_test.py

```diff
@@ -0,0 +1,94 @@
+    'datacatalog_metadata_ingestor.DataCatalogMetadataIngestor.__init__',
+    lambda self, *args: None)
+@mock.patch('google.datacatalog_connectors.commons.cleanup.'
+            'datacatalog_metadata_cleaner.DataCatalogMetadataCleaner.__init__',
+            lambda self, *args: None)
+@mock.patch('google.datacatalog_connectors.commons.monitoring.'
+            'metrics_processor.MetricsProcessor.__init__',
+            lambda self, *args: None)
+class DatacatalogSynchronizerTestCase(unittest.TestCase):
+    __MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+    __KAKFA_PACKAGE = 'google.datacatalog_connectors.kafka'
+    __COMMONS_PACKAGE = 'google.datacatalog_connectors.commons'
+    __PROJECT_ID = 'test_project'
+    __LOCATION_ID = 'location_id'
+    __HOST = 'localhost'
+    __ENTRY_GROUP_ID = 'kafka'
+
+    @mock.patch('google.datacatalog_connectors.kafka.'
+                'scrape.metadata_scraper.MetadataScraper.get_metadata')
+    @mock.patch('google.datacatalog_connectors.kafka.'
+                'prepare.assembled_entry_factory.'
+                'AssembledEntryFactory.make_entries_from_cluster_metadata')
+    @mock.patch('google.datacatalog_connectors.commons.ingest.'
+                'datacatalog_metadata_ingestor.'
+                'DataCatalogMetadataIngestor.ingest_metadata')
+    @mock.patch('google.datacatalog_connectors.commons.cleanup.'
+                'datacatalog_metadata_cleaner.DataCatalogMetadataCleaner.'
+                'delete_obsolete_metadata')
+    @mock.patch('google.datacatalog_connectors.commons.monitoring.'
+                'metrics_processor.MetricsProcessor.'
+                'process_elapsed_time_metric')
+    @mock.patch('google.datacatalog_connectors.commons.monitoring.'
+                'metrics_processor.MetricsProcessor.'
+                'process_metadata_payload_bytes_metric')
+    @mock.patch('google.datacatalog_connectors.commons.monitoring.'
+                'metrics_processor.MetricsProcessor.'
+                'process_entries_length_metric')
+    def test_synchronize_metadata_should_not_raise_error(  # noqa: E125
+            self, process_entries_length_metric,
+            process_metadata_payload_bytes_metric, process_elapsed_time_metric,
+            delete_obsolete_metadata, ingest_metadata,
+            make_entries_from_cluster_metadata, get_metadata):
+        make_entries_from_cluster_metadata.return_value = [{}]
+
+        synchronizer = datacatalog_synchronizer.DataCatalogSynchronizer(
+            DatacatalogSynchronizerTestCase.__PROJECT_ID,
+            DatacatalogSynchronizerTestCase.__LOCATION_ID,
+            DatacatalogSynchronizerTestCase.__ENTRY_GROUP_ID,
+            DatacatalogSynchronizerTestCase.__HOST,
+            test_utils.FakeMetadataScraper,
+            {'bootstrap.servers': 'test_address'},
+            enable_monitoring=True)
+
+        synchronizer.run()
+        self.assertEqual(get_metadata.call_count, 1)
```

Comment:
those assertions don't seem to be checking if error was raised, but rather if some methods were called. I think test's name should reflect what is being checked; an alternative would be to add a line of comment before the test explaining that.